endpoint/spec.md

```diff
@@ -33,7 +33,11 @@ this form:
   "name": "STRING", ?
   "description": "STRING", ?
   "documentation": "URL", ?
-  "labels": { "STRING": "STRING" * }, ?
+  "labels": {
+    "modelversion": "1.0",
+    "compatiblewith": "https://github.com/xregistry/spec/blob/main/endpoint/spec.md",
```

Comment:
This feels a bit odd... sort of like saying "I'm compatible with myself".
Is this trying to define the "compatiblewith" value that extensions models are meant to use?

Reply:
Think of it like `"openApi": "3.1"` in OpenAPI or `"asyncAPI": "2.1"` in AsyncAPI. 

Then vendors like MSFT or SAP can create their own extensions of such standards, but there they will also point out that it's still a compatible e.g., OpenAPI 3.1 document.

Somehow you need to state that this a document follows a model and that the model is actually an interoperable standard. And yes, the base model is compatible with itself, but "compatiblewith" sounds a bit unusual.

Reply:
Right, so I would think that in endpoint's model.json the label name might be something like "self" instead since we're not really trying to claim any kind of compatibility statement. But it then provides the value that extension models can use in their "compatibilewith" label... if they don't know where it lives. Or am I not following?

Reply:
Yes, it's indeed a "self" statement, but "compatible" is also true. If you deploy that model as is, it's basically compatible with the standard model, it just doesn't have any extensions and modifications. 

But I wonder if when you import such a model, you want to keep this "compatiblewith" statement by default. If it's not there, you have no guarantuee that you understand that model in someone elses xRegistry.